modules/database/src/Database.ts

```diff
@@ -80,18 +82,26 @@ export default class DatabaseModule extends ManagedModule {
   }
 
   async onServerStart() {
-    await this._activeAdapter.createSchemaFromAdapter(models.DeclaredSchema);
-    // Introspection takes place
+    const isConduitDB = await this._activeAdapter.isConduitDB();
+    if (isConduitDB) {
+      await this._activeAdapter.createSchemaFromAdapter(models.DeclaredSchema);
+    } else {
+      await this.introspectDB();
```

Comment:
The reason why this check was added was so that we could create `DeclaredSchema` **after** schema introspection to prevent `isConduitDB()` from failing on an interrupted introspection.
Tbh this still wouldn't completely prevent a half-way introspection as it could crash after the schemas have been introspected and before they have been stored.
After adding `PendingSchemas`, `introspectDB()` ended up creating it anyway, therefore: 

- `DeclaredSchema` should always be created at the top
- `isConduitDB()` should be smarter (eg check if a core schema exists as `Core` only registers these after the initial introspection is over), 
- If `!isConduitDB()`, run `introspectDB()`, creating `PendingSchemas` etc